README.md

```diff
@@ -224,7 +224,7 @@ mindex.updateRecord({
 
 | key | default | type | description |
 | --- | --- | --- | --- |
-| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not compound a string or integer may be used. |
+| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not a compound object, a string or integer may be used. |
```

Comment:
The index can either be compound or not. I would not say that the index itself is a compound object.

The index being compound means that it is indexed by multiple keys.

Can you remove this change from your PR?